nbgrader/tests/apps/test_api.py

```diff
@@ -21,6 +21,7 @@ def api(request, course_dir, db, exchange):
     config = Config()
     config.CourseDirectory.course_id = "abc101"
     config.Exchange.root = exchange
+    config.Exchange.cache = exchange
```

Comment:
Usually this should be a different directory... it's probably fine here but just for consistency you probably want to pass the "cache" fixture to the signature of this method and then set that as the path here (which should just automagically work, because `cache` is defined in `conftest.py` and pytest is magic):

```
def api(request, course_dir, db, exchange, cache):
    ...
    config.Exchange.cache = cache
    ...
```

Reply:
Wasn't actually sure if that would just work and forgot to test it out in the end. Will change that.